--- qce-ga-0.40c/quickcam.c~	Thu Mar  4 08:43:04 2004
+++ qce-ga-0.40c/quickcam.c	Thu Mar  4 08:43:04 2004
@@ -34,6 +34,7 @@
 */
 
 #include <linux/config.h>
+#include <asm/current.h>
 #include <linux/kernel.h>
 #include <linux/init.h>
 #include <linux/sched.h>
--- qce-ga-0.40c/quickcam.h~	Thu Mar  4 09:03:12 2004
+++ qce-ga-0.40c/quickcam.h	Thu Mar  4 09:03:12 2004
@@ -1,6 +1,7 @@
 #ifndef __LINUX_QUICKCAM_H
 #define __LINUX_QUICKCAM_H
 
+#include <asm/current.h>
 #include <linux/list.h>
 #include <linux/usb.h>
 #include <linux/videodev.h>
--- qce-ga-0.40c/memory.c~	Thu Mar  4 11:42:45 2004
+++ qce-ga-0.40c/memory.c	Thu Mar  4 11:42:45 2004
@@ -19,6 +19,7 @@
  *
 */
 
+#include <asm/current.h>
 #include <linux/wrapper.h>
 #include <linux/usb.h>
 #include <asm/io.h>
